src/common/cluster-store.ts

```diff
@@ -11,9 +11,10 @@ import { appEventBus } from "./event-bus"
 import { dumpConfigYaml } from "./kube-helpers";
 import { saveToAppFiles } from "./utils/saveToAppFiles";
 import { KubeConfig } from "@kubernetes/client-node";
+import { subscribeToBroadcast, unsubscribeAllFromBroadcast } from "./ipc";
 import _ from "lodash";
 import move from "array-move";
-import { subscribeToBroadcast, unsubscribeAllFromBroadcast } from "./ipc";
+import type { WorkspaceId } from "./workspace-store";
```

Comment:
You already importing it as non-type module (see on top), so no need to define extra here.

Reply:
If not imported separately as a `type`, webpack warnings will appear in compilation time.